google/google.go

```diff
@@ -91,6 +91,10 @@ func (c *Client) updateCreds(token *oauth2.Token) {
 }
 
 func (c *Client) HasCreds() bool {
+	if c.credentials == nil {
```

Comment:
I looked for a way around these nil checks that shouldn't be necessary but it's not straightforward so will just go with it for now.